tests/pyincore/analyses/galvestoncge/test_galvestoncge.py

```diff
@@ -0,0 +1,26 @@
+from pyincore import IncoreClient, Dataset
+from pyincore.analyses.galvestoncge import GalvestonCGEModel
+import pyincore.globals as pyglobals
+
+# This script runs SaltLakeCGEModel analysis with input files from
+# IN-CORE development services.
+
+
+def run_base_analysis():
+    client = IncoreClient(pyglobals.INCORE_API_DEV_URL)
+    galveston_cge = GalvestonCGEModel(client)
+
+    galveston_cge.set_parameter("model_iterations", 1)
+    galveston_cge.load_remote_input_dataset("SAM", "63c8398f4b843238326be519")
+    galveston_cge.load_remote_input_dataset("BB", "63a354f54b8432383268cd92")
+    galveston_cge.load_remote_input_dataset("MISCH", "63c838954b843238326be513")
+    galveston_cge.load_remote_input_dataset("EMPLOY", "63c836d94b843238326bd982")
+    galveston_cge.load_remote_input_dataset("JOBCR", "63c837f74b843238326be507")
+    galveston_cge.load_remote_input_dataset("OUTCR", "63c838414b843238326be50d")
+    galveston_cge.load_remote_input_dataset("sector_shocks", "63a358404b8432383268cee0")
+
+    galveston_cge.run_analysis()
```

Comment:
Make sure you replace those ids with 
```
641e1fccea308903b2b8c51a SAM
641e1fff34810d74880b3985 BB
641e209eea308903b2b8c51b JOBCR
641e20f134810d74880b3986 OUTCR
641e2152ea308903b2b8c51c Employ
641e21a702414a24a5436dfe MISCH
```

It won't work for dev yet but eventually we will migrate the dataset to dev and those will be the id

Reply:
Those ids are still Salt lake city ids btw